From b657b01f6ecb6a5f65039f13b74b5c16762ea9a2 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 11 Mar 2023 15:38:55 +0100
Subject: [PATCH] CMakeLists.txt: use libproc2 from procps-4

* there might be many more changes needed to adapt to procps 4 API, see:
  https://gitlab.com/procps-ng/procps/-/issues/239

* procps upgrade was now merged in oe-core together with igt-gpu-tools adaptation in:
  https://git.openembedded.org/openembedded-core/commit/?id=631eba02d64f1a42514e0ae4361bbecc5cce5fa7

Upstream-Status: Pending

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9ffa77a..2c2d690 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,7 @@ find_package(Boost REQUIRED COMPONENTS regex system filesystem)
 include_directories(${Boost_INCLUDE_DIRS})
 webos_add_compiler_flags(ALL ${Boost_CFLAGS_OTHER})
 
-pkg_check_modules(PROCPS REQUIRED libprocps)
+pkg_check_modules(PROCPS REQUIRED libproc2)
 include_directories(PROCPS_INCLUDE_DIRS)
 webos_add_compiler_flags(ALL ${PROCPS_CFLAGS})
 
